boa_engine/src/vm/opcode/swap/mod.rs

```diff
@@ -23,3 +23,40 @@ impl Operation for Swap {
         Ok(ShouldExit::False)
     }
 }
+
+/// `Swap3` implements the Opcode Operation for `Opcode::Swap3`
+///
+/// Operation:
+///  - Swap the top two values on the stack.
```

Comment:
Is this comment accurate? Looks like it’s swapping index 0 and index 2

Reply:
> Is this comment accurate? Looks like it’s swapping index 0 and index 2

It isn't. This is PR is still a WIP so I haven't updated all the documentations. I will update after the refactor is complete.